dev/merge_arrow_pr.py

```diff
@@ -278,8 +299,11 @@ def show(self):
 
 
 def get_candidate_fix_version(mainline_versions,
-                              merge_branches=('master',),
+                              merge_branches=None,
                               maintenance_branches=()):
+    if merge_branches is None:
+        merge_branches = (git_default_branch_name(),)
```

Comment:
Thanks @lafiona for the changes! Only one more thing, now `merge_branches` is never used, we can get rid of it and we can also get rid of the new function created to retrieve the `git_default_branch_name` because the script is never calling that function once we remove `merge_branches`. That makes sense as we use the GitHub API to merge the PR and we use the base branch defined on the PR, not the one on this script.

Reply:
Hi @raulcd, I apologize that I didn't see this earlier, it's tied to the feedback that you already left yesterday! And I've removed the input argument as well as references to `merge_branches` in the function body. Thanks for pointing it out!

Reply:
Thank you very much! I'll wait for the linter to pass and I'll merge after successful build